benchmarks/random_allocations/random_allocations.cpp

```diff
@@ -310,7 +310,7 @@ int main(int argc, char** argv)
       std::string mr_name = args["resource"].as<std::string>();
       declare_benchmark(mr_name);
     } else {
-      std::array<std::string, 4> mrs{"pool", "binning", "cuda", "arena"};
+      std::array<std::string, 4> mrs{"pool", "binning", "arena", "cuda"};
```

Comment:
This is an unrelated change. It just moves "cuda" last since it takes forever to run.